content/doc/developer/plugin-development/choosing-jenkins-baseline.adoc

```diff
@@ -5,24 +5,37 @@ title: Choosing a Jenkins version to build against
 
 Your plugin's POM controls which version of Jenkins it is building/testing against.
 
-## Choosing a version
+== Choosing a version
 
 You need to balance compatibility and features:
 
-* *Keeping the Jenkins version your plugin builds against low* will allow more users to install and use your plugin. 
+* *Keeping the Jenkins version your plugin builds against low* will allow more users to install and use your plugin.
 In particular, the LTS Release Line is based on slightly older releases to provide a more stable experience for conservative users.
-* *Building against recent Jenkins versions* allows you to use recently added core features and API from your plugin. 
-You will also use that newer version for `mvn hpi:run`, so it may be easier to test your plugin with newer Jenkins releases. 
+* *Building against recent Jenkins versions* allows you to use recently added core features and API from your plugin.
+You will also use that newer version for `mvn hpi:run`, so it may be easier to test your plugin with newer Jenkins releases.
 Additionally, since features are sometimes moved from core into plugins, depending on a more recent Jenkins version will make your plugin's dependencies on what used to be core features explicit, otherwise your plugin will not work.
-* *Use at least the minimum version supported by the update center*, the supported update center versions can be found at link:https://updates.jenkins.io[updates.jenkins.io], at the time of writing (2020-03) this is 2.164.x.
+* *Use at least the minimum version supported by the update center*, the supported update center versions can be found at link:https://updates.jenkins.io[updates.jenkins.io], at the time of writing (August 2020) this is 2.176.x.
```

Comment:
This has the problem that it's not sure that these are always starting at .1, so this contradicts advice further down. Dynamic tiers do not go by baselines, but individual releases.

Reply:
> This has the problem that it's not sure that these are always starting at .1, so this contradicts advice further down.

Which advice? I think a plugin developer will be smart enough to select the .2 version if there is no .1 version.

Maybe it would help that we explicitly set the recommended baseline version when we select the new LTS baseline. That means this document (or another one?) needs to change with every LTS minor version.

Reply:
I am referring to this advice:

> When setting the Jenkins version please also make sure to use the .1 patch version of the LTS version: 2.204.1, or 2.222.1.

The way update-center2 currently works is that it only looks at the release date and goes back 1 year, so the oldest supported release there will not be a .1 release.

Of course this also says "at least" and if we consider these version requirements ("not older than", "only .1") as a sort of "Venn diagram" then the area of overlap will be the reasonable choices.

But I do wonder whether we can rewrite this page to present all of these considerations, and perhaps even make the versions we consider good choices explicit. Can probably even be automated, especially once https://github.com/jenkins-infra/update-center2/pull/424 gives access to `tiers.json`.

Reply:
I would say at least use the oldest line supported by the `bom`…so you can _use_ the BOM, and keep it fresh with Dependabot.